## Problem Statements:

### A F&B manager wants to determine whether there is any significant difference in the diameter of the cutlet between two units. A randomly selected sample of cutlets was collected from both units and measured? Analyze the data and draw inferences at 5% significance level. Please state the assumptions and tests that you carried out to check validity of the assumptions.


### Import libraries

In [1]:
import pandas as pd
from scipy import stats

### Reading the csv file

In [7]:
cutlets=pd.read_csv("Cutlets.csv")
cutlets.head()

Unit A  Unit B
0  6.8090  6.7703
1  6.4376  7.5093
2  6.9157  6.7300
3  7.3012  6.7878
4  7.4488  7.1522

### H0 - There is no significant difference in the diamaters of unit A and unit B (diameteres of the cutlets are same)
### H1 - There is a significant difference in the diamaters of unit A and unit B (diameteres of the cutlets are not same)

### Performing a two tail T Test, to check the hypothesis we made

In [21]:
stats.ttest_ind(cutlets["Unit A"],cutlets["Unit B"])

Ttest_indResult(statistic=0.7228688704678061, pvalue=0.4722394724599501)

## Conclusion:
### Pvalue = 0.47>0.05. So, we accept H0. 
### There is no significant difference in the diamaters of unit A and unit B (diameteres of the cutlets are same)